py3status/parse_config.py

```diff
@@ -348,6 +363,31 @@ def make_value_from_env(self, value):
         except ValueError:
             self.error('Bad conversion')
 
+    def make_value_from_exec(self, value):
+        conversion_options = {
+            'int': int,
+            'float': float,
+            'str': str,
+
+            'bool': (lambda val: val.lower() in ('', 'true', '1')),
+            'auto': self.make_value,
+        }
+
+        match = re.match('exec\((.+?)(\s*,\s*(int|float|str|bool|auto))?\)', value)
+        shell, _, var_type = match.groups()
+        try:
+            shell_stdout = str(check_output(shell, shell=True).rstrip(), encoding='utf-8')
```

Comment:
FWIW, we can't bypass  `rstrip()` here. If necessary, we could add `tr -d '[:space:]'` or such in `exec()`.